bundles/core/org.eclipse.smarthome.core.thing/src/main/java/org/eclipse/smarthome/core/thing/util/ThingHandlerHelper.java

```diff
@@ -15,20 +15,30 @@
  * This class provides utility methods related to the {@link ThingHandler} class.
  *
  * @author Markus Rathgeb - Initial contribution
+ * @author Simon Kaufmann - added UNKNOWN
  */
 public class ThingHandlerHelper {
 
     private ThingHandlerHelper() {
     }
 
     /**
+     * Checks if the given state indicates that a thing handler has been initialized.
+     *
+     * @return true if the thing handler has been initialized, otherwise false.
+     */
+    public static boolean isHandlerInitialized(final ThingStatus thingStatus) {
```

Comment:
It seems to be required [here](https://github.com/eclipse/smarthome/pull/2612/files#diff-6d6897e04b52e8efb8749b37d50acbdeR299), a place where before the check against ONLINE/OFFLINE was done. And I do not see how we could solve it differently.